game/templates/game/board.html

```
@@ -15,26 +15,346 @@
     <script>
         window.si = window.si || function () { (window.siq = window.siq || []).push(arguments); };
     </script>
-    <script defer src="/_vercel/speed-insights/script.js"></script>
+
+    {% comment %} <script defer src="/_vercel/speed-insights/script.js"></script> {% endcomment %}
+
+    <style>
+
+        /* ============================================================
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </style></head></html> ], open tag match failed [ <style> ] on line 21.

(tag-pair)